extensions/wikia/PortableInfobox/tests/PortableInfoboxDataServiceTest.php

```diff
@@ -109,7 +164,9 @@ public function testImageListRemoveDuplicates() {
 	public function testImageListFetchImages() {
 		$images = PortableInfoboxDataService::newFromTitle( $this->prepareTitle( 1 ) )
 			->purge()
-			->setPagePropsProxy( new PagePropsProxyDummy( [ '1infoboxes' => json_encode( $this->getInfoboxData() ) ] ) )
+			->setPagePropsProxy(
+				new PagePropsProxyDummy( [ '1infoboxes' => json_encode( $this->getInfoboxPageProps() ) ] )
+			)
 			->getImages();
 
 		$this->assertEquals( [ 'Test.jpg', 'Test2.jpg' ], $images );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Test2.jpg" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)